### ISSUE TITLE
Project Dead?

### ISSUE THREAD

**Issue by marcussacana** [lllyasviel/style2paints#149]
I was worderning, a good time without news since the last v4.5 beta, I was really hyped to see the v5 release but we don't even have a date as far I know.
Our god, @lllyasviel, you don't have any news to us mortals?

**Comment by SylveonBottle** [comment]
Seems like Clip Studio Paint integrated something like this into their program.

https://www.youtube.com/watch?v=waifOEGnH-c

**Comment by lllyasviel** [comment]
In the past year, I was learning new things in the field of computer graphics and mathematical imaging. 

In 2019 autumn, I found deep learning far from good enough for professional use cases and began to learn 2D graphical techniques. 

I also made several attempts to train new models but finally I find the 4.5 model is already the best and it is not very likely that I can get improvements via deep learning, so computer graphics and mathematical imaging become important.

The project is not dead. I have a twitter account and usually update my research progress on it.
